src/styles/blocks/stars.scss

```
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
```

use the power of nesting
```suggestion

  &--1 .stars__star:nth-child(-n + 1),
  //...
}
```